xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -599,6 +599,7 @@ void CPeripheralCecAdapter::OnTvStandby(void)
   case LOCALISED_ID_STOP:
     KODI::MESSAGING::CApplicationMessenger::GetInstance().PostMsg(TMSG_MEDIA_STOP);
     break;
+  case LOCALISED_ID_IGNORE:
```

Comment:
In the current state of this PR, the message (`Unexpected [standby_pc_on_tv_standby] setting value`) will still be logged, as this case falls through do the default case and the `CLog::Log(...)` function call.

Reply:
lol @rechi beat me by 7 minutes. This needs 
```suggestion
  case LOCALISED_ID_IGNORE:
  break;
```

Reply:
You are right. I just added the break.